src/index.html

```diff
@@ -1,13 +1,376 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>MyBike</title>
+    <link rel="stylesheet" href="./styles/style.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="icon" href="./images/logo.svg">
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="/" class="logo logo--white">
+              <img 
+                src="images/logo.svg"
+                alt="MyBIKE white logo"
+                class="logo__image"
+              >
+            </a>
+
+            <div class="header__links">
+              <a href="tel:+1 234 555-55-55" class="icon icon--phone">
```

Comment:
You need to change format phone only in href. Check it everywhere
```suggestion
              <a href="tel:+12345555555" class="icon icon--phone">
```